Detectors/ITSMFT/MFT/workflow/src/TrackerSpec.cxx

```diff
@@ -427,5 +430,13 @@ DataProcessorSpec getTrackerSpec(bool useMC, int nThreads)
     Options{}};
 }
 
+///_______________________________________
+TrackerDPL::~TrackerDPL()
+{
+  // Deallocate the memory that was previously reserved for these arrays.
```

Comment:
also here the mutex should be used to avoid race condition:
```
std::lock_guard<std::mutex> guard(o2::mft::TrackerConfig::sTCMutex);
TrackerConfig::mBins.reset();
TrackerConfig::mBinsS.reset();
```

Reply:
Ok thanks! So I keep the destructor here in `TrackerSpec.cxx` even if I move the `initBinContainer` to `Tracker.cxx`?

Reply:
Ah, no, sorry, overlooked that. This should be done in the destructor of the Tracker. The trackers will be destructed by the automatic destructor of the TrackerSpec.

Reply:
Ok I will do it